WalletWasabi.Gui/Config.cs

```diff
@@ -84,6 +86,24 @@ public class Config : ConfigBase
 		[JsonConverter(typeof(EndPointJsonConverter), Constants.DefaultRegTestBitcoinP2pPort)]
 		public EndPoint RegTestBitcoinP2pEndPoint { get; internal set; } = new IPEndPoint(IPAddress.Loopback, Constants.DefaultRegTestBitcoinP2pPort);
 
+		[DefaultValue(false)]
+		[JsonProperty(PropertyName = "JsonRpcServerEnabled", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public bool JsonRpcServerEnabled { get; internal set; }
+
+		[DefaultValue("")]
+		[JsonProperty(PropertyName = "JsonRpcUser", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public string JsonRpcUser { get; internal set; }
```

Comment:
I think we remove rpcuser altogether.

Reply:
Also do we need rpc password and user in the first place? It's not like anyone who gets access to the computer wouldn't be able to read it.

Reply:
That's true. I would like to be able to publish this through TLS when dotnet core allows me to do so in the future but without that this makes few sense except for those that try to connect to the rpc server from other machines. 

If we remove this then we have to prevent access from other machines (what could be a good idea)